### DISCUSSION TITLE
Plans for using edition 2024

### DISCUSSION THREAD

**Discussion by OneOfOne** [tokio-rs/tokio#6680] [Ideas]
Mainly for https://rust-lang.github.io/rfcs/3498-lifetime-capture-rules-2024.html

**Comment by mox692** [comment]
FYI, the last edition update was done when tokio's msrv was above the 2021 edition release version(1.56).
https://github.com/tokio-rs/tokio/pull/5559

**Comment by Darksonn** [comment]
What msrv is required for this?

**Reply by OneOfOne** [reply]
v1.82.0

**Reply by Darksonn** [reply]
Okay. That will be a long time.

Which Tokio functions do you expect this to affect? It's not clear to me that we have any.
